packages/quantic/force-app/main/default/lwc/quanticRecommendationList/__tests__/quanticRecommendationList.test.js

```diff
@@ -0,0 +1,428 @@
+/* eslint-disable jest/no-conditional-expect */
+/* eslint-disable no-import-assign */
+// @ts-ignore
+import QuanticRecommendationList from 'c/quanticRecommendationList';
+import {cleanup, flushPromises, buildCreateTestComponent} from 'c/testUtils';
+import * as mockHeadlessLoader from 'c/quanticHeadlessLoader';
+
+const recommendationsArray = [
+  {
+    title: 'Title 1',
+    clickUri: 'https://example.com/1',
+    excerpt: 'Excerpt 1',
+  },
+  {
+    title: 'Title 2',
+    clickUri: 'https://example.com/2',
+    excerpt: 'Excerpt 2',
+  },
+  {
+    title: 'Title 3',
+    clickUri: 'https://example.com/3',
+    excerpt: 'Excerpt 3',
+  },
+  {
+    title: 'Title 4',
+    clickUri: 'https://example.com/4',
+    excerpt: 'Excerpt 4',
+  },
+];
```

Comment:
Is there any particular reason why we need 4 recommendations?
If yes fine!
If no, we can reduce the number and and make the test a bit more simple.

Reply:
yes originally it was because the default recommendationsPerRow is 3 so I wanted to test with 1 more. But then I realized that we are not testing the display of rows here since this is handled by the carousel and the grid so I can reduce it to like 2 recommendations :)